infra/gcp/clusters/projects/kubernetes-public/aaa/10-cluster-configuration.tf

```
@@ -154,11 +154,11 @@ resource "google_container_cluster" "cluster" {
     resource_limits {
       resource_type = "cpu"
       minimum = 2
-      maximum = 16
+      maximum = 200
```

is it possible to have :
```suggestion
      maximum = 64
```

---

Arnaud, the limit is for the entire cluster, not a single machine. Technically we can use 64, but this is relatively close to the cluster size we use today (~26 cores). Why do we want to put this limit low?

---

From my perspective, we increase the limits for only 1 workload. Most of the workloads actually running don't need a lot of CPU. I think we should for the moment double the current limit and adjust again when needed. 
If the limits I suggest is not enough for your workloads I'm happy to remove my suggestions.